src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -3294,14 +3294,14 @@ namespace InternalHeatGains {
                         ZoneOtherEq(Loop).OtherEquipFuelType = noOtherFuelTypeZero;
                         FuelTypeString = AlphaName(2);
                     } else {
-                        ExteriorEnergyUse::ValidateFuelType(ZoneOtherEq(Loop).OtherEquipFuelType,
+                        ExteriorEnergyUse::ValidateFuelType(state.exteriorEnergyUse, ZoneOtherEq(Loop).OtherEquipFuelType,
```

Comment:
This is really nice.  I can see a not-so-distant future where this line changes from:

```
ExteriorEnergyUse::ValidateFuelType(state.exteriorEnergyUse, ...);
```
to
```
state.exteriorEnergyUse.validateFuelType(...);
```